.circleci/config.yml

```diff
@@ -911,6 +968,32 @@ jobs:
             docker push nucypher/nucypher:$CIRCLE_TAG
             docker push nucypher/nucypher:latest
 
+  publish_porter_docker:
+    working_directory: ~/nucypher
+    docker:
+      - image: cimg/python:3.8.6
+    steps:
+      - checkout
+      - setup_remote_docker
+      - restore_cache:
+          keys:
+            - v1-{{ .Branch }}-{{ arch }}
+          paths:
+            - ~/docker/nucypher-porter.tar
+      - run:
+          name: Load Porter Docker Image Layer Cache
+          command: |
+            set +o pipefail
+            docker load -i ~/docker/nucypher-porter.tar | true
+      - deploy:
+          name: Push Tagged NuCypher Porter Docker Images
+          command: |
+            echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
+            docker tag nucypher/nucypher-porter:circle nucypher/nucypher-porter:$CIRCLE_TAG
+            docker tag nucypher/nucypher-porter:circle nucypher/nucypher-porter:latest
+            docker push nucypher/nucypher-porter:$CIRCLE_TAG
+            docker push nucypher/nucypher-porter:latest
```

Comment:
Seems a bit redundant, why not `nucypher/porter?`

Reply:
✔️